shared/src/mobileMain/kotlin/dev/johnoreilly/confetti/SpeakerDetailsViewModel.kt

```diff
@@ -27,7 +27,7 @@ class SpeakerDetailsViewModel(
 
     val speaker: StateFlow<UiState> = flow {
         // FixMe: add .speaker(id)
-        val response = repository.conferenceData(conference = conference, FetchPolicy.CacheOnly)
+        val response = repository.conferenceData(conference = conference, FetchPolicy.CacheFirst)
```

Comment:
CacheFirst seems safer and more like what we want than CacheOnly.

Reply:
+1. That'll fix a deepling to a speaker details if the conference has never been opened before.
Also we should get only the speaker instead of retrieving all the conference data